Microsoft.Toolkit.Wpf.UI.XamlHost/WindowsXamlHostWrapperConverter.cs

```diff
@@ -2,39 +2,39 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
```

Comment:
looks like this is only whitespace changes, so it should probably be reverted (same with OSVersionHelper.cs)

Reply:
Its CR/LF changes, the files in git are using UNix style and VS is transform to Windows style.

---
In reply to: [258750369](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/81#discussion_r258750369) [](ancestors = 258750369)

Reply:
yeah, it's an unfortunate world we live in :/